spark/v3.3/spark/src/main/java/org/apache/iceberg/spark/SparkSQLProperties.java

```diff
@@ -42,4 +44,9 @@ private SparkSQLProperties() {}
   // Controls whether to check the order of fields during writes
   public static final String CHECK_ORDERING = "spark.sql.iceberg.check-ordering";
   public static final boolean CHECK_ORDERING_DEFAULT = true;
+
+  // Controls whether case of schema field name needs to be considered during case sensitive
+  // operations
```

Comment:
I'm still not sure why we are making this an Iceberg Property here. I think we should probably just refer to it via SQLConf in the code below. If we want to keep this then we need to fix the doc here since the line wrap seems to have put another endline after operations.